docs/operations/comanage-troubleshooting-guide.md

```diff
@@ -26,6 +26,26 @@ We have seen this occur if a user clicks the confirmation link then closes the t
 1.  Now that the necessary identifiers exist for the `CO Person` record, the LDAP DN can be computed and the record
     provisioned in LDAP. To make sure, click on `Provisioned Services` and then  `Provision`.
 
+### The COManage petition is stuck in the "Approved" state
+
+This may happen when the approver has internet issues while attempting to approve a CO Petition.
+
+1.  Navigate to the Petitioner's CO Person page (this can be done directly from the CO Petition under the `Attched Identities` section).
+
+1.  Confirm that the CO Person is missing identifiers of types `UID` and `OSG ID`.
+
+1.  On the CO Person page, confirm that the CO Person's status and Role status are both `Approved`.
+
+1.  On the broken CO Person page, click `Autogenerate Identifiers` on the right side of the page.
```

Comment:
I think steps 2 + 4 here can be combined into one:

"If the CO Persion is missing identifiers..., click `Autogenerate Identifiers`..."